source/administration/config-settings.rst

```diff
@@ -994,16 +994,31 @@ Note that the gossip port and gossip protocol are used to determine cluster heal
 Enable Experimental Gossip Encryption
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-**True**: All communication through the cluster using the gossip protocol will be encrypted.
+**True**: All communication through the cluster uses the gossip protocol will be encrypted.
```

Comment:
I'm not a grammar expert, but "using" seems to have been the correct word? :thinking:

Reply:
You're absolutely right, @agnivade. I'll get these corrected.